robot/Cumulus/resources/NPSP.py

```diff
@@ -1179,8 +1179,7 @@ def change_view_to(self,view_name):
         self.selenium.wait_until_page_contains(view_name)
 
     def search_field_by_value(self, fieldname, value):
-         """ Populate field with Place Holder as a locator
-             and actual value of the place holder.
+         """ search the field with the placeholder given by 'fieldname' for the given 'value'
```

Comment:
Probably rephrase it to 'Searches the field with the given "fieldname" & "value"' ?

Reply:
done